src/main/java/io/snyk/jenkins/tools/SnykInstaller.java

```diff
@@ -177,6 +179,7 @@ public Platform call() {
   }
 
   private static class Downloader extends MasterToSlaveCallable<Void, IOException> {
+    @Serial
```

Comment:
Java 17 is now the supported baseline for Jenkins. `@Serial` is a Java 17 feature (introduced in Java 14). More details: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/Serial.html